css/styles.css

```diff
@@ -0,0 +1,367 @@
+}
+
+.carousel__button{
+    position: absolute;
+    top: 50%;
+    transform: translateY(-50%);
+    border: none;
+    background-color: rgba(0, 0, 0, 0.3);
+    color: var(--color-white);
+    cursor: pointer;
+}
+
+.carousel__button--left{
+    left: -2.5rem;
+}
+
+.carousel__button--right{
+    right: -2.5rem;
+}
+
+.carousel__nav{
+    display: flex;
+    justify-content: center;
+    padding: 10px 0;
+}
+
+.carousel__indicator{
+    border: 0;
+    border-radius: 50%;
+    width: 0.8rem;
+    height: 0.8rem;
+    background: rgba(0, 0, 0, 0.3);
+    margin: 0 12px;
+    cursor: pointer;
+}
+
+.carousel__indicator--active{
+    background: var(--color-dark);
+}
+
+.row-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 0 auto;
+    width: 80%;
+}
+
+/*Image banner */
+.banner {
+    background-image: url("https://th.bing.com/th/id/R.7ed07dd0533c0e6e1ae94b82f9c0e233?rik=Zap9cIYL%2f1qnzg&riu=http%3a%2f%2fwww.pixelstalk.net%2fwp-content%2fuploads%2f2016%2f06%2fGirl-Fashion-Dress-Red-Background-HD-Wallpaper.jpg&ehk=QJZWVgeD0Rnc8jWUbIlgrg6SGetjVrbcTpjeN4iRJRg%3d&risl=&pid=ImgRaw&r=0");
+    width: 100%;
+    height: 60vh;
+    background-position: top 4rem center;
+    background-attachment: fixed;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    color: var(--color-white);
+    padding: 5rem;
+}
+
+/* Brand section */
+.brands {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.brands>* {
+    height: 13rem;
+    width: 15rem;
+    object-fit: contain;
+}
+
+/* Customer reviews */
+
+.testimonials{
+margin-bottom: 4rem;
+}
+
+.testimonials > .card{
+    width: 16rem;
+    padding: 1rem;
+    font-size: 1rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+
+.testimonial__profile{
+    margin-bottom: 1rem;
+}
+
+.testimonial__content{
+    /* padding: 1rem 0; */
+}
+
+.testimonial__content > h4{
+    text-align: center;
+}
+
+.testimonials .socials{
+    cursor: pointer;
+}
+
+/* footer */
+footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: var(--spacing-large);
+    background-color: rgba(var(--color-complementary-values), 0.1);
+    color: var(--color-complementary);
+}
+
+.code-symbol {
+    font-size: var(--fsize-400);
+    font-weight: var(--fweight-bold);
+    color: var(--color-primary);
+}
+
+.footer__list {
+    list-style: none;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 1rem;
+}
+
+.footer__list .fa {
+    font-size: 1.3rem;
+    color: rgba(var(--color-complementary-values), 0.8);
+}
+
+.footer__list .fa:hover {
+    color: var(--color-primary);
+    transition: color 0.3s ease-in;
+}
+
+.copyright {
+    font-size: var(--fsize-200);
+    color: rgba(var(--color-complementary-values), 0.8);
+}
+
+@media screen and (min-width: 2000px) {
```

Comment:
If possible, varialize media screen values, instead of 2000px can use variables for ease of management in longer run.